### PR TITLE
git-local-commits: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

File: pages/common/git-local-commits.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# git local-commits
+
+> Show local commits which haven't been pushed to origin.
```
Comment (bl-ue) [inline pages/common/git-local-commits.md:3]: Seems like a useful extra bit of info to me:
```suggestion
> Show local commits which haven't been pushed to origin. Any additional arguments will be passed directly to `git log`.
```

**Review by bl-ue — APPROVED** [review]
LGTM!

File: pages/common/git-local-commits.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# git local-commits
+
+> Show local commits which haven't been pushed to origin. Any additional arguments will be passed directly to `git log`.
+> Part of `git-extras.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-local-commits>.
+
+- Show commits haven't been pushed:
```
Comment (nicokosi) [inline pages/common/git-local-commits.md:7]:
```suggestion
- Show commits that have not been pushed:
```

Comment (CleanMachine1) [reply pages/common/git-local-commits.md:7]: What about 
```suggestion
- Show commits that haven't been pushed:
```

Comment (bl-ue) [reply pages/common/git-local-commits.md:7]: Either are fine with me.  When and when not to use contractions can be tricky 🤷🏻‍♂️

Comment (CleanMachine1) [reply pages/common/git-local-commits.md:7]: Having the contraction makes more sense in this case. I have commited it

File: pages/common/git-local-commits.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# git local-commits
+
+> Show local commits which haven't been pushed to origin. Any additional arguments will be passed directly to `git log`.
```
Comment (bl-ue) [inline pages/common/git-local-commits.md:3]: I think that sounds slightly more fluent, and it's what's used in the example.
```suggestion
> Show local commits that haven't been pushed to origin. Any additional arguments will be passed directly to `git log`.
```

Comment (CleanMachine1) [reply pages/common/git-local-commits.md:3]: Doesn't matter to me however I have no objections. Committed

**Review by marchersimon — APPROVED** [review]
Looks good

**Comment by CleanMachine1** [comment]
Can I merge?

**Comment by marchersimon** [comment]
Maybe we should wait for @nicokosi's approval, since they requested changes.

**Comment by CleanMachine1** [comment]
Give it another 2 days then i am merging, is that ok?

In the meantime @nicokosi you are wanted here! :smile:

**Review by nicokosi — APPROVED** [review]
👍

**Pull request merged**